### ISSUE TITLE
State not received when remote is pressed.

### ISSUE THREAD

**Issue by Scriptman** [Scriptman/ESPHome_ITHO_Eco_Fan_CC1101#6]

**Comment by asterixonline** [comment]
Wow, if this works to, i'll be verry happy! I tryed the https://github.com/svollebregt/ESPEASY_Plugin_ITHO code first but that one didn't work for my fan. So if you manage to get the remote state in the code I'll be very happy!

**Comment by Scriptman** [comment]
I tried to merge that code with this repo, but that didn't work. I'm debugging the code, because the first time it worked, so I think i changed something and broke it

**Comment by Dijkkie** [comment]
The script is working with the C1101, only the receive part of the remote not yet. I have check the ID.  And fill it in the scripts, only the status will not be changed at Home Assistant. I have CVE-S ECO SE with remote control. Is there a workaround to let it work.

**Comment by Scriptman** [comment]
> The script is working with the C1101, only the receive part of the remote not yet. I have check the ID. And fill it in the scripts, only the status will not be changed at Home Assistant. I have CVE-S ECO SE with remote control. Is there a workaround to let it work.

You are right, working on the receiving part for some time, without any luck.
